### PR TITLE
update full-quiz.js

### PR DESCRIPTION
**Pull request by Abdullah-Irfan-web**
Added python questions in full-quiz.js

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,4 +1,41 @@
...
+  { 
+    Question:
+     "What is a correct syntax to output "Hello World" in Python?",
+    Answer: "print("Hello World")",
+    Distractor1: "echo("Hello World")",
+    Distractor2: "p("Hello World")",
+    Distractor3: "printf("Hello World")",
+    Distractor4: "echo(Hello World)",
+    Explanation:
+    "In Python 2, there was an internal limit to how large an integer value could be. But that limit was removed in Python 3.",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:36]: I think the explanation needs to be changed so it relates more to the question.

Proposed fix: To print anything in Python, you use the print() function.

File: src/data/full-quiz.js [hunk 2]
```diff
@@ -1,4 +1,41 @@
 const fullQuiz = [
+  
+   {
+   Question:
+     "Which operator is used to find remainder in Python?",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:5]: We already have a question similar to this one. 
Please remove this question

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @Abdullah-Irfan-web ! 

Thank you for your quiz questions. 
Please see me comments to make the necessary changes before it gets merged into the main branch. 

Thanks! 👍

File: src/data/full-quiz.js [hunk 3]
```diff
@@ -1,4 +1,41 @@
...
+    Distractor3: "create function_name(parameters):",
+    Distractor4: "none",
+    Explanation:
+    "The following snippet shows the general syntax to define a function in Python: def function_name(parameters):",
+    Link: "https://www.freecodecamp.org/news/functions-in-python-a-beginners-guide/",
+  },
+  { 
+    Question:
+     "What is a correct syntax to output "Hello World" in Python?",
+    Answer: "print("Hello World")",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:30]: For the answer and distractors please use single quotes for Hello World

Proposed fix: 'Hello World'

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]

**Comment by Abdullah-Irfan-web** [comment]
Okay i will fix those changes asap

File: src/data/full-quiz.js [hunk 4]
```diff
@@ -2,38 +2,36 @@ const fullQuiz = [
...
-    Distractor1: "*",
-    Distractor2: "X",
-    Distractor3: "/",
-    Distractor4: "#",
+     "Which method can be used to replace parts of a string?",
+    Answer: "replace()",
+    Distractor1: "switch()",
+    Distractor2: "repl()",
+    Distractor3: "find()",
+    Distractor4: "replacString()",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:10]: Please delete distractor4

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
I think everything looks good.
You just need to remove that extra distractor4 answer and we can merge it.

Remember that there will always only be a total of 4 answers (3 distractors and 1 correct answer) 👍

File: src/data/full-quiz.js (hunk 4 above)
Comment (Abdullah-Irfan-web) [reply src/data/full-quiz.js]: I have done the
Thank you so much for your guidance

**Review by jdwilkin4 — APPROVED** [review]
LGTM! 👍

**Comment by jdwilkin4** [comment]
Hi @Abdullah-Irfan-web ! 
Congrats on your first PR for the Developer Quiz Site.
Your questions will also be used in the Learn to Code RPG game.
We look forward to future contributions.

**Pull request merged**